src/modules/Login/action.js

```diff
@@ -279,6 +285,10 @@ export const mergeSettings = (
         // for each currency info (each native currency)
         const pluginDenominations = account.currencyConfig[currencyName].currencyInfo.denominations // get denominations for that plugin
         const settingDenominationIndex = pluginDenominations.findIndex(pluginDenom => pluginDenom.multiplier === loadedSettings[key].denomination) // find settings denom in plugin denoms
+        if (loadedSettings[key].multiplier !== pluginDenominations[0].multiplier) finalSettings[key].multiplier = pluginDenominations[0].multiplier
```

Comment:
This logic looks like it would reset all user settings, so if I set my BTC wallet to "bits", it would always go back to BTC. Actually, the surrounding code already does the right thing - it checks to be sure the user setting is one of the valid options, and only resets it if not.